python/GafferImageUI/ImageViewUI.py

```diff
@@ -614,21 +615,28 @@ def __eventPosition( self, event ):
 	def __buttonPress( self, ui, event ) :
 
 		if event.buttons == event.Buttons.Left and not event.modifiers :
+			self.__createInspectorStartPosition = None
 			return True # accept press so we get dragBegin() for dragging color
 		elif event.buttons == event.Buttons.Left and event.modifiers == GafferUI.ModifiableEvent.Modifiers.Control :
-			self.__dragStartPosition = self.__eventPosition( event )
+			self.__createInspectorStartPosition = self.__eventPosition( event )
 			self.addInspector()
 			ci = self.getPlug().parent().children()[-1]
-			Gaffer.Metadata.registerValue( ci["pixel"], "__hovered", True )
-			ci["pixel"].setValue( self.__dragStartPosition )
+			Gaffer.Metadata.registerValue( ci["pixel"], "__hovered", True, persistent = False )
+			ci["pixel"].setValue( self.__createInspectorStartPosition )
 
 			return True # creating inspector
 		else:
 			return False
 
+	def __buttonRelease( self, ui, event ) :
+		if self.__createInspectorStartPosition:
```

Comment:
Because other handlers could accepts `buttonPress()` (for instance, if the Viewer decided to add a right-click menu), it's possible we could get here without `self.__createInspectorStartPosition` ever being assigned to. Might be best to assign `self.__createInspectorStartPosition = None` in our constructor?

Reply:
Addressed in 35ac605